src/shared/core/BackgroundScriptUpdateHandler.ts

```diff
@@ -75,7 +75,7 @@ export class BackgroundScriptUpdateHandler {
     const number = getRandomInteger();
     return Promise.race([
       sendPortMessage<{ syn: number }, { ack: number }>(port, { syn: number }),
-      rejectAfterDelay(1000),
+      rejectAfterDelay(1000, 'sendPortMessage'),
```

Comment:
I suggest adding user-readable errors everywhere. At least, error screen will look more friendly:))

Reply:
I think it's quite hard to come up with "user readable" messages for each request. What would you suggest?

Reply:
Background port is unavailable

Reply:
I am not sure this is more user readable 😁